hypnos/prj.conf

```diff
@@ -49,7 +49,7 @@ CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_ATT_ENFORCE_FLOW=n
 
 # Apparently necessary for bluetooth
-CONFIG_HEAP_MEM_POOL_SIZE=16384
+#CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
Heap allocation should be avoided in embedded systems. It put constraints on the memory and is not viable in the long run.
I did not find much features that needs Heap (filesystem, DMA for STM32 and a display driver (but not the pinetime's display driver). And as soon as you avoid the call `malloc` & Co. you are good to go without heap.

Reply:
I had copied this from a Bluetooth sample but things seem to work fine without it so I’ll remove it.